common/http/platform/beast/http.cpp

```diff
@@ -235,28 +235,34 @@ Client::Client(
 	const ClientConfig &client, events::EventLoop &event_loop, const string &logger_name) :
 	event_loop_ {event_loop},
 	logger_name_ {logger_name},
+	http_proxy_ {client.http_proxy},
+	https_proxy_ {client.https_proxy},
+	no_proxy_ {client.no_proxy},
```

Comment:
`no_proxy_` is not used anywhere, it's only set here.

Reply:
Very good catch! This is indeed a bug. And a good example of why unit tests != integration tests.